<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="soccf-tr5-001" name="component-tests(failed)" tests="2" failures="1" timestamp="7 Aug 2014 06:44:31 GMT" time="17.146" errors="0">
  <testcase name="testValidate" time="0.452" classname="teammates.test.cases.common.EvaluationAttributesTest"/>
  <testcase name="testAll" time="4.462" classname="teammates.test.cases.logic.FeedbackSessionsLogicTest">
    <failure type="com.google.apphosting.api.ApiProxy$CallNotFoundException" message="The API package &amp;apos;datastore_v3&amp;apos; or call &amp;apos;RunQuery()&amp;apos; was not found.">
      <![CDATA[com.google.apphosting.api.ApiProxy$CallNotFoundException: The API package 'datastore_v3' or call 'RunQuery()' was not found.
	at com.google.apphosting.api.ApiProxy$1.get(ApiProxy.java:170)
	at com.google.apphosting.api.ApiProxy$1.get(ApiProxy.java:168)
	at com.google.appengine.api.utils.FutureWrapper.get(FutureWrapper.java:86)
	at com.google.appengine.api.utils.FutureWrapper.get(FutureWrapper.java:86)
	at com.google.appengine.api.datastore.FutureHelper.getInternal(FutureHelper.java:71)
	at com.google.appengine.api.datastore.FutureHelper.quietGet(FutureHelper.java:32)
	at com.google.appengine.api.datastore.QueryResultsSourceImpl.peekQueryResultAndIfFirstRecordIndexList(QueryResultsSourceImpl.java:167)
	at com.google.appengine.api.datastore.QueryResultsSourceImpl.loadMoreEntities(QueryResultsSourceImpl.java:109)
	at com.google.appengine.api.datastore.QueryResultsSourceImpl.loadMoreEntities(QueryResultsSourceImpl.java:92)
	at com.google.appengine.api.datastore.QueryResultIteratorImpl.ensureLoaded(QueryResultIteratorImpl.java:145)
	at com.google.appengine.api.datastore.QueryResultIteratorImpl.hasNext(QueryResultIteratorImpl.java:64)
	at org.datanucleus.store.appengine.query.RuntimeExceptionWrappingIterator$1.get(RuntimeExceptionWrappingIterator.java:50)
	at org.datanucleus.store.appengine.query.RuntimeExceptionWrappingIterator$1.get(RuntimeExceptionWrappingIterator.java:46)
	at org.datanucleus.store.appengine.query.QueryExceptionWrappers$1.get(QueryExceptionWrappers.java:51)
	at org.datanucleus.store.appengine.query.QueryExceptionWrappers$2.get(QueryExceptionWrappers.java:86)
	at org.datanucleus.store.appengine.query.RuntimeExceptionWrappingIterator.hasNext(RuntimeExceptionWrappingIterator.java:105)
	at org.datanucleus.store.appengine.query.LazyResult.resolveAll(LazyResult.java:115)
	at org.datanucleus.store.appengine.query.LazyResult.size(LazyResult.java:110)
	at org.datanucleus.store.appengine.query.StreamingQueryResult.size(StreamingQueryResult.java:130)
	at org.datanucleus.store.query.AbstractQueryResult.isEmpty(AbstractQueryResult.java:312)
	at teammates.storage.api.CoursesDb.getCourseEntity(CoursesDb.java:151)
	at teammates.storage.api.CoursesDb.getEntity(CoursesDb.java:138)
	at teammates.storage.api.EntitiesDb.deleteEntities(EntitiesDb.java:213)
	at teammates.logic.backdoor.BackDoorLogic.deleteCourses(BackDoorLogic.java:452)
	at teammates.logic.backdoor.BackDoorLogic.deleteExistingData(BackDoorLogic.java:434)
	at teammates.test.cases.BaseTestCase.removeAndRestoreDatastoreFromJson(BaseTestCase.java:155)
	at teammates.test.cases.logic.FeedbackSessionsLogicTest.testGetFeedbackSessionResultsSummaryAsCsv(FeedbackSessionsLogicTest.java:1123)
	at teammates.test.cases.logic.FeedbackSessionsLogicTest.testAll(FeedbackSessionsLogicTest.java:103)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- testAll -->
</testsuite> <!-- component-tests(failed) -->
